ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/TransientQueryResourceCleanerIntTest.java

```diff
@@ -75,10 +75,11 @@ public class TransientQueryResourceCleanerIntTest {
     private static final String USER_TABLE = USER_DATA_PROVIDER.sourceName();
 
     // Persistent Topics:
+    // _confluent-command, // This topic is created by ce-kafka for LicenseStore
     // _confluent-ksql-default__command_topic,
     // PAGEVIEW_TOPIC,
     // USER_TOPIC
-    private static final int numPersistentTopics = 3;
+    private static final int numPersistentTopics = 4;
```

Comment:
This won't pass unless we use ce kafka right?

Reply:
Yes. ce-kafka creates a topic internally called `_confluent-command`